package-lock.json

```diff
@@ -24785,7 +24785,7 @@
       "integrity": "sha512-L6rvG9GDxaLgFjg41K+5Yv9OMrU98sWe+Ykmc6FDJW/+vYZMhdOMKkISgzptMaERHvS2Y2lw9MDRm2gHhlQQoA==",
       "dev": true,
       "requires": {
-        "regenerator-runtime": "0.13.1"
+        "regenerator-runtime": "^0.13.4"
```

Comment:
is there a reason the package version changed? I thought it was pinned to 0.13.1

Reply:
Oh I think master doesn't have the npm overrides changes yet, and is still built using npm 6. For develop we probably don't want to change these versions though?

Reply:
Running it in develop has the same result, i'm getting invalid for the 0.13.4 version and deduped for 0.13.1 so I think it's ok?
<img width="901" alt="Screen Shot 2022-03-31 at 5 22 24 PM" src="https://user-images.githubusercontent.com/36055303/161151488-599aa759-25e6-4849-837a-31c6c128b5bc.png">
Would you all prefer if I revert package-lock (except the SDK version change) to merge this in? We can make a separate item to dig into this maybe

Reply:
sure that works for me

Reply:
The "no unsafe eval CSP" percy snapshot still looks correct so we have some protection
https://percy.io/cd93112c/yext-answers/builds/16917527/search/preview/950769252?browser=firefox&searchParam=unsafe&viewLayout=side-by-side&viewMode=new&width=1280